src/windows/Button.tsx

```diff
@@ -45,6 +47,11 @@ export class Button extends ButtonBase implements React.ChildContextProvider<But
 
     private _isFocusedWithKeyboard = false;
 
+    //Offset to show context menu using keyboard.
+    protected _contextMenuOffset() {
+        return {x: 0, y: 0};
```

Comment:
Nit: there should be extra spaces within the curly brackets for consistency.

Reply:
updated